packages/utilities/src/utilities/scrollIntoView/scrollIntoView.js

```diff
@@ -0,0 +1,86 @@
+/**
+ * Copyright IBM Corp. 2016, 2020
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+/**
+ * The viewport calculation for 12.5vh + 32px of the root margins.
+ *
+ * @private
+ */
+const _viewportMargin =
+  '-' +
+  (
+    Math.round(document.documentElement.clientHeight / (100 / 12.5)) + 32
+  ).toString() +
+  'px 0px';
+
+/**
+ * Intersection Observer options
+ *
+ * @private
+ */
+const _options = {
+  rootMargin: _viewportMargin,
+  threshold: 0,
+};
+
+/**
+ * Utility handles fade transition for selected elements.
+ *
+ * @example
+ * import '@carbon/ibmdotcom-styles/scss/internal/scroll-into-view/_scroll-into-view.scss';
+ * import { scrollIntoView } from '@carbon/ibmdotcom-utilities';
+ *
+ * As an example, the function can be called to target '.bx--content-block' as such:
+ *
+ * For default values of 400ms and continuous play:
+ * scrollIntoView('.bx--content-block')
+ *
+ * With 'one and done' option:
+ * scrollIntoView('.bx--content-block', false)
+ *
+ * For custom delay time, set within targeted class in the application's CSS code as such:
+ *
+ * .bx--content-block {
+ *   --#{$dds-prefix}--scroll-into-view-delay: 3s;
+ * }
+ *
+ * @param {*} selector menu item selector id
+ * @param {boolean} iterations to define whether its continuous or not
+ */
+
+const scrollIntoView = (selector, iterations = true) => {
```

Comment:
Mere question, but will this only allow for a string selector, or can it pass in a Node object?

Reply:
Currently it's set to receive strings, mainly since it's wrapped around a listener that waits until the whole page is loaded before querying the element. If the adopters can make sure the element exists before calling the function we could make it so they can pass node objects instead.